mrjob/cloud.py

```diff
@@ -119,14 +121,13 @@ def __init__(self, **kwargs):
         # store the (tunneled) URL of the job tracker/resource manager
         self._ssh_tunnel_url = None
 
-
-
     ### Options ###
 
     def _default_opts(self):
         return combine_dicts(
             super(HadoopInTheCloudJobRunner, self)._default_opts(),
             dict(
+                cloud_upload_part_size=100,  # 100 MB
```

Comment:
cloud_upload_part_size_mb ?

Reply:
Good point. I think `cloud_part_size_mb` would do it, though that's outside the scope of bringing Dataproc up to parity with EMR. Filed this as #1774.